### ISSUE TITLE
Innotop does not read /root/.my.cnf when containerized

### ISSUE THREAD

**Issue by meersjo** [innotop/innotop#199]
I'm building a container for Innotop; and am having trouble with what should be the default behaviour of reading  `~/.my.cnf` which I'm mounting in to `/root/.my.cnf`.
```
$ podman run -it --rm -v /etc/my.cnf:/root/.my.cnf:ro [registry]/innotop
localhost PROCESSLIST_NO_IS: DBI connect(';host=localhost;mysql_read_default_group=client','',...) failed: Can't connect to local server through socket '/run/mysqld/mysqld.sock' (2) at /usr/local/bin/innotop line 7777.
```
It's worth noting that the `.my.cnf` I l mount in contains not only user and password, but also the host to target, because the mysql socket itself is of course not visible from the innotop container (and the mysql itself is also in one). I know that that default `host=localhost` in the DSN from the error message will make DBI try the socket; but I feel like the `host=` in the `.my.cnf` should be overriding that.

After some debugging inside the container, I have found that when I create a very simple `/root/.innotop/innotop.conf` file with nothing but what should be default options, it works as expected. This is the `innotop.conf`:
```
[general]
[connections]
default=dsn=DBI:mysql:;mysql_read_default_group=client
```

This is my Dockerfile:
```
FROM alpine:3.19
  ARG INNOTOP_VERSION=1.13.0
  ARG INNOTOP_SOURCE=https://github.com/innotop/innotop/archive/refs/tags/v${INNOTOP_VERSION}.tar.gz

WORKDIR /tmp
  RUN apk add perl perl-dbd-mysql perl-term-readkey perl-time-hires build-base
  RUN wget ${INNOTOP_SOURCE}
  RUN tar xvzf v${INNOTOP_VERSION}.tar.gz

WORKDIR /tmp/innotop-${INNOTOP_VERSION}
  RUN perl Makefile.PL
  RUN make install

WORKDIR /root/.innotop
  RUN echo '[general]'                                               > innotop.conf
  RUN echo '[connections]'                                          >> innotop.conf
  RUN echo 'default=dsn=DBI:mysql:;mysql_read_default_group=client' >> innotop.conf

WORKDIR /
  RUN rm -rf /tmp/*
  RUN rm -rf /var/cache/apk/*
  RUN apk del build-base

ENTRYPOINT ["/usr/local/bin/innotop"]
```

**Comment by yoku0825** [comment]
Thank you for using innotop.

I think this behavior doesn't depend on container or native linux box.

When innotop couldn't find any DSN in the `~/.innotop`, use `host=localhost` forcefully ( then your host and port configuration in `/root/.my.cnf` are ignored, because DSN host=localhost override them)

https://github.com/innotop/innotop/blob/master/innotop#L7950

**Comment by yoku0825** [comment]
I will test #200 in my environment.
Please try it if you can.

**Comment by meersjo** [comment]
Confirm that patch works as expected, thank you.

**Comment by yoku0825** [comment]
Fixed by #200 .
Thank you for using innotop.
